code/game/machinery/doors/airlock.dm

```diff
@@ -540,7 +540,7 @@
 			if(!(unres_sides & heading))
 				continue
 			var/image/floorlight = image(icon='icons/obj/doors/airlocks/station/overlays.dmi', icon_state="unres_[heading]")
-			floorlight.plane = ABOVE_LIGHTING_PLANE
+			SET_PLANE_EXPLICIT(floorlight, ABOVE_LIGHTING_PLANE, src)
```

Comment:
the painful part is if this was a mutable appearance, and used its construct proc, this would have been caught

Reply:
Yep! I believe images are used here due to airlocks also being emitter blockers, so we have this nasty manual image.

Reply:
I uh, it shouldn't actually matter, it's used as an overlay and under the hood the two are very similar
If you want to be very based, convert it to mutable appearances usin the proc and all

Reply:
I'll be honest, I assumed it was an image and not a `mutable_appearance` for some intentional hacky reaason, but yep works great as a mutable appearance. Changed.